regression/cases/splunk_hec_route_s3/vector/vector.toml

```diff
@@ -34,8 +34,10 @@ address = "0.0.0.0:9090"
 
 [sinks.s3_sidecar]
 type = "aws_s3"
+acknowledgements.enabled = true
```

Comment:
The current soaks don't have this configuration. Did you find it to be needed?

Reply:
We believe that the soak analysis code is masking a bug here. The lading configuration has acks flipped on, meaning if `ackId` isn't present in the payload it's an error. Lading's behavior is unchanged between the soaks and the regression detector here. The new analysis code fails with an error, specifically, that no bytes were written by lading in enough intervals to cause a problem. The old analysis code shuffles this under the rug.

Considering that lading was configured to ack I figured that Vector must have been intended to do the same. But, now that I'm writing this I have not confirmed the ack/no-ack state of things in the soak setup, which I will do presently.

Reply:
Oh well look at that. In #14935 I think we missed the fact that https://github.com/vectordotdev/vector/blob/0e686508e604c7e7fdb4e20737b15ad425c825fd/soaks/tests/splunk_hec_route_s3/lading.yaml is using the http generator instead of the splunk_hec generator specifically, so the intention is flipped from what I have here: no-ack.

Reply:
@jszwedko addressed in [c256ed7](https://github.com/vectordotdev/vector/pull/15063/commits/c256ed76abdbeeafe0721b11833f19b5dc1b7785).